_data/simple-icons.json

```diff
@@ -6186,6 +6186,16 @@
             "hex": "AC2E45",
             "source": "https://commons.wikimedia.org/wiki/File:MTR_(logo_with_text).svg"
         },
+        {
+            "title": "MUI",
+            "aliases": {
+                "aka": [
+                    "Material-UI"
+                ]
+            },
+            "hex": "007FFF",
+            "source": "https://github.com/mui-org/material-ui/blob/master/docs/src/icons/SvgMuiLogo.tsx"
```

Comment:
Can you use the permalink here?
https://github.com/mui-org/material-ui/blob/353cecb5391571163eb6bd8cbf36d2dd299aaf56/docs/src/icons/SvgMuiLogo.tsx

Reply:
Thanks for the correction - I updated the source link

Reply:
It looks like you commited Jest Cache folder. Can you remove it?

Reply:
Removed